src/Sulu/Bundle/ContentBundle/DocumentManager/ContentInitializer.php

```diff
@@ -28,29 +28,30 @@
 class ContentInitializer implements InitializerInterface
 {
     /**
-     * @var DocumentRegistry
+     * @var ConnectionRegistry
      */
-    private $registry;
+    private $connectionRegistry;
 
     /**
      * @var string
      */
     private $languageNamespace;
 
-    public function __construct(ConnectionRegistry $registry, $languageNamespace)
+    public function __construct(ConnectionRegistry $connectionRegistry, $languageNamespace)
     {
-        $this->registry = $registry;
+        $this->connectionRegistry = $connectionRegistry;
         $this->languageNamespace = $languageNamespace;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function initialize(OutputInterface $output)
+    public function initialize(OutputInterface $output, $purge = false)
     {
-        $workspace = $this->registry->getConnection()->getWorkspace();
-        $this->initializeNamespaces($output, $workspace);
-        $this->initializeContentTypes($output, $workspace);
+        foreach ($this->connectionRegistry->getConnections() as $connection) {
```

Comment:
This change means that every workspace is purged and initialized, not only the default one. This change shouldn't make any difference for current installations, since we always only have one workspace, but will make a difference as soon as #2361 is merged, because we will have separate workspaces for draft and public.